examples/api/lib/material/tooltip/tooltip.1.dart

```diff
@@ -37,7 +38,7 @@ class TooltipSample extends StatelessWidget {
       ),
       height: 50,
       padding: const EdgeInsets.all(8.0),
-      preferBelow: false,
+      preferBelow: true,
```

Comment:
There's a bit that needs to be updated for this in the doc that points to this example:
```dart
 /// `preferBelow` is false, the tooltip will prefer showing above [Tooltip]'s child widget.
 /// However, it may show the tooltip below if there's not enough space
 /// above the widget.
```